android/pytorch_android/src/androidTest/java/org/pytorch/PytorchTestBase.java

```diff
@@ -367,16 +384,106 @@ public void testChannelsLastConv2d() throws IOException {
         outputNCHW,
         MemoryFormat.CONTIGUOUS,
         new long[] {1, 3, 2, 2},
-        new long[] {2, 4, 6, 8, 11, 12, 13, 14, -101, -102, -103, -104});
+        new long[] {
+          2*111, 2*112,
+          2*121, 2*122,
+
+          211, 212,
+          221, 222,
+
+          -311, -312,
+          -321, -322});
 
     final IValue outputNHWC =
         module.runMethod("conv2d", IValue.from(inputNHWC), IValue.from(wNHWC), IValue.from(true));
     assertIValueTensor(
         outputNHWC,
         MemoryFormat.CHANNELS_LAST,
         new long[] {1, 3, 2, 2},
-        new long[] {2, 11, -101, 4, 12, -102, 6, 13, -103, 8, 14, -104});
-  }
+        new long[] {
+          2*111, 211, -311,      2*112, 212, -312,
+          2*121, 221, -321,      2*122, 222, -322});
+  }
+
+  // @Test
```

Comment:
why is this test commented out?

Reply:
I'll split it, thx.

Reply:
Would you please review those ? 
https://github.com/pytorch/pytorch/pull/114790
https://github.com/pytorch/pytorch/pull/114791
https://github.com/pytorch/pytorch/pull/114792
https://github.com/pytorch/pytorch/pull/114793
https://github.com/pytorch/pytorch/pull/114794
https://github.com/pytorch/pytorch/pull/114798